backend/packages/Upgrade/commands/banner.ts

```diff
@@ -1,12 +1,12 @@
-import chalk from 'chalk';
 import figlet from 'figlet';
 
 figlet.text(process.argv[2], (error: any, data: any) => {
   if (error) {
     return process.exit(1);
   }
-
-  console.log(chalk.blue(data));
-  console.log('');
-  return process.exit(0);
+  import('chalk').then((chalk) => {
+    chalk.default.blue(data);
+    console.log('');
+    return process.exit(0);
+  });
```

Comment:
`figlet.text()` can return `void`. For me the banner worked if I changed the above to the following:
```
if (error) {
    process.exit(1);
  } else {
    import('chalk').then((chalk) => {
      console.log(chalk.default.blue(data));
      process.exit(0);
    });
  }
```